### PR TITLE
Reset button

### PR DESCRIPTION
**Pull request by olliebrownlow**
Also does a touch of styling

### CODE REVIEW

File: src/web/index.html [hunk 1]
```diff
@@ -15,13 +15,13 @@
 
     <div id="live-chart-page">
       <div id="displaying-reading" style="visibility: hidden">Your plant's current soil moisture reading:
-        <span id="reading"></span>
-        <br>
-        <span id="moisture-text"></span>
+        <span id="reading"></span><br>
+        <span id="moisture-text">placeholder text</span>
```
Comment (laurencetaylor) [inline src/web/index.html:19]: Do we need placeholder text?

**Review by laurencetaylor — COMMENTED** [review]
Looks good

File: src/web/index.html (hunk 1 above)
Comment (olliebrownlow) [reply src/web/index.html:19]: Only to keep the gap when it's initially hidden. If there is another way to do this we can change it.

**Review by breycarr — APPROVED** [review]
Very clear functionality and easy to read, no amendments needed.

**Pull request merged**